arangod/Cluster/ClusterMethods.cpp

```diff
@@ -1213,6 +1213,8 @@ futures::Future<OperationResult> countOnCoordinator(
     reqOpts.timeout = network::Timeout(10.0);
   }
 
+  network::addUserParameter(reqOpts, trx.username());
```

Comment:
For future us: We could try to prevent code duplication here, as this patterns repeats.